lib/strategies/simple.js

```diff
@@ -76,10 +92,16 @@ class SimpleConcat {
     }
 
     let entry = {
-      file: file,
-      content: content
+      file: file
     };
 
+    if (this._isContentLarge(content)) {
+      entry.content = '';
```

Comment:
lets do null or undefined here, that way we can disambiguate between empty content and no content.